### PR TITLE
Fix Cors so that api and client can communicate with each other

### PR DESCRIPTION
**Pull request by christerforsgren91**
PT https://www.pivotaltracker.com/story/show/178002471

## User stories
```
As a developer
In order to deploy my app
I need to setup Cors
```

## Changes proposed

- Setup Cors

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -101,6 +101,8 @@ GEM
     minitest (5.14.4)
     msgpack (1.4.2)
     nio4r (2.5.7)
+    nokogiri (1.11.3-x86_64-darwin)
```
Comment (LucasKnudsen) [inline Gemfile.lock:104]: Get out of here with your darwin

File: config/initializers/cors.rb [hunk 2]
```diff
@@ -5,12 +5,13 @@
...
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
+  allow do
+    origins '*'
+
+    resource '*',
+      headers: :any,
+      expose: %w[access-token expiry token-type uid client],
+      methods: [:get, :post, :put, :patch, :delete, :options, :head]
+  end
+end
```
Comment (LucasKnudsen) [inline config/initializers/cors.rb]: You guys already configured CORS in `application.rb`. 
Choose which place you want to configure it. 

(I'd go with this, deleting what you have in application.rb)

File: Gemfile.lock (hunk 1 above)
Comment (christerforsgren91) [reply Gemfile.lock:104]: noo i need my darwin!!

File: config/initializers/cors.rb (hunk 2 above)
Comment (christerforsgren91) [reply config/initializers/cors.rb]: aha okey, we fix that!

**Pull request merged**